uploader/api_commands.py

```diff
@@ -67,11 +67,22 @@ def upload_video():
     # Call celery
     procesar_video.apply_async(args=args, queue='batch_videos')
 
-
     return jsonify(
         id=video.id,
         message='File uploaded successfully'
     )
 
+
+@app.route('/api-queries/uploader/video', methods=['GET'])
+def video():
+    video_id = request.args.get('id')
+    if video_id is not None:
+        video = db.session.query(Video).filter(Video.id == video_id).first()
+        return video_schema.dump(video)
+    else:
+        videos = db.session.query(Video).all()
+        return videos_schema.dump(videos)
```

Comment:
Debemos dividir esto en 2 getters diferentes :
1. recuperar la información de una tarea en la aplicación:
    -  Aqui retornamos la información de un video.
    - Si el video no existe, se retorna un 404 not found 
2.  recuperar todas las tareas de edición de un usuario 
    - max (int). Parámetro opcional que filtra la cantidad de resultados de una consulta.  
    - order (int). Especifica si los resultados se ordenan de forma ascendente (0) o de forma descendente (1) según el ID de la tarea. 
    - Si no hay tareas se retorna una lista vacía

Reply:
Hecho en el último commit.